### PR TITLE
Added the feature to determine the object within 10cm

### PR DESCRIPTION
**Pull request by iamanshulgit**

### CODE REVIEW

File: main.ino [hunk 1]
```diff
@@ -38,6 +47,28 @@ void setup() {
...
+  digitalWrite(trigPin,LOW);
+  
+  //Measuring the duration of the reflecting sound
+  duration = pulseIn(echoPin, HIGH);
+
+  //Calculte the distance(in cm) from the object
+  distance = (duration/2)/29.1;
+  
+  //display the duration and dis
+  Serial.print(distance);
```
Comment (mayankofficial999) [inline main.ino:65]: These lines are not needed in this function
They are already available in the calcDistance() function in the latest master branch.

File: main.ino [hunk 2]
```diff
@@ -38,6 +47,28 @@ void setup() {
...
+
+  //returning 1 if object is less than 10cm, else return 0
+  if (distance <= 10 )
+    {return 1;}
+   else
+    {return 0;}
```
Comment (mayankofficial999) [inline main.ino:71]: - These lines are enough for the function.

Comment (iamanshulgit) [reply main.ino:71]: If you want I can remove it.

**Comment by himanshushah05** [comment]
Hi @iamanshulgit, we have added some comments to your code. Please resolve them.
Kindly refer to the latest code in the master branch which got updated recently.
We simply need the function to check if an obstacle is present using the distance variable passed in the argument and not calculating the distance itself in the function.

**Comment by iamanshulgit** [comment]
Thank you for guiding me.

**Comment by iamanshulgit** [comment]
Hello @mayankofficial999 
I hope this code is working this time. Thank you.

File: main.ino [hunk 3]
```diff
@@ -1,19 +1,28 @@
 // Define the pins used to control
+
+// Defines variables used
+
+int distance; // variable for the distance measurement
```
Comment (mayankofficial999) [inline main.ino:5]: - This line is not needed

Comment (mayankofficial999) [reply main.ino:5]: - Just make this small change and PR is good to merge !

Comment (iamanshulgit) [reply main.ino]: I have removed the distance varible. Can you please check? Thank you

**Comment by mayankofficial999** [comment]
@iamanshulgit Congratulations ! Your PR got merged. Thanks for your contribution.

**Pull request merged**